docs/source/command_line.rst

```diff
@@ -168,7 +168,11 @@ imports.
 
 .. option:: --follow-untyped-imports
 
-    This flag makes mypy analyze imports without stubs or a py.typed marker.
+    This flag makes mypy analyze imports even if missing a py.typed marker or stubs.
```

Comment:
Might be good to explicitly mention that this is about installed packages, including those installed with pip? For example, something along these lines:
> This flag makes mypy analyze imported installed packages (including third-party packages installed using pip), even if they don't explicitly declare support for type checking by providing a py.typed marker or stubs.

Maybe also add a link from "py.typed marker" to relevant external documentation?